lxd/documentation.go

```diff
@@ -58,11 +58,10 @@ func metadataConfigurationGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
-	var data map[string]any
-	err = json.Unmarshal(file, &data)
+	formattedJSON, err := json.MarshalIndent(json.RawMessage(file), "", "  ")
 	if err != nil {
 		return response.SmartError(err)
 	}
 
-	return response.SyncResponse(true, data)
+	return response.SyncResponsePlain(true, true, string(formattedJSON))
```

Comment:
this changes the response output format doesn't it?

Reply:
So here how it was before:
![before-metadata](https://github.com/canonical/lxd/assets/27967361/2e0acb0e-f53a-4d0c-817b-dfb50b9e3a76)


And here how it is with the change:
![after-metadata](https://github.com/canonical/lxd/assets/27967361/2ef482ee-cbf3-4e5a-9a19-2e3ded9787ee)

Reply:
I don't exactly know how this API endpoint is consumed but I'd assume that a Javascript client has no problem dealing with that. Correct me @edlerd if I'm wrong. But this way is more human readable is gzip encoded.

Reply:
Yes, I am confident the JSON parser will ignore the extra whitespace.

Personally, I am fine with or without the extra formatting. I always use the network tab of the browser to explore JSON responses. There the structure is parsed and presented in a nice format.

Reply:
I don't think `SyncResponsePlain` adds the `content-type` header with `appliation/json` which this endpoint probably should have.

Reply:
fixed